src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -29,6 +31,24 @@
   (fn []
     (utils/show-popup (i18n/label :t/transaction-moved-title) (i18n/label :t/transaction-moved-text))))
 
+(re-frame/reg-fx
+  ::show-transaction-error
+  (fn [message]
+    ;; (andrey) we need this timeout because modal window conflicts with alert
+    (js/setTimeout #(utils/show-popup (i18n/label :t/transaction-failed) message) 1000)))
+
+(re-frame/reg-fx
+  :discard-transaction
+  (fn [id]
+    (status/discard-transaction id)))
+
+;;Helper functions
+
+(defn transaction-valid? [{{:keys [to data]} :args}]
+  (or (and to (utils.hex/valid-hex? to)) (and data (not= data "0x"))))
```

Comment:
Is `web3.isAddress` not better here?

Reply:
tbh idk why do we need this functions at all here, i just moved it, it wasn't added in this PR